integration/spark/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -921,52 +921,55 @@ object CommonUtil {
     var i = 0
     val fieldTypesLen = fields.length
     while (i < fieldTypesLen) {
-      if (!row.isNullAt(i)) {
-        fields(i).dataType match {
-          case StringType =>
-            data(i) = DataTypeUtil.getBytesDataDataTypeForNoDictionaryColumn(row.getString(i),
+      fields(i).dataType match {
```

Comment:
I think initially, the if check of null at row was there for all datatypes, now handled for string, other data types will be handled?

Reply:
It is already handled for other types..Test cases is added